subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -672,12 +670,57 @@ static int bearer_ctl_send_unacked(struct net_buf *buf, void *user_data)
 	prov_clear_tx();
 	k_work_reschedule(&link.prot_timer, PROTOCOL_TIMEOUT);
 
-	bt_mesh_adv_send(buf, &buf_sent_cb, user_data);
+	bt_mesh_adv_send(buf, NULL, NULL);
 	net_buf_unref(buf);
 
 	return 0;
 }
 
+static void bearer_ctl_send_link_close(void)
+{	/*
+	 * According to mesh profile spec (5.3.1.4.3), the close message should
+	 * be restransmitted at least three times.
+	 */
+	struct net_buf *buf =
+		ctl_buf_create(LINK_CLOSE, &link.close_link.reason, 1, RETRANSMITS_LINK_CLOSE);
+
+	if (!buf) {
+		LOG_ERR("Failed to create link close buffer");
+		return;
+	}
+
+	prov_clear_tx();
+	k_work_reschedule(&link.prot_timer, PROTOCOL_TIMEOUT);
```

Comment:
Should it be started only once? And is there any need in it if `protocol_timeout` doesn't do anything if link is not `ADV_LINK_ACTIVE`?